ee/desktop/runtime/runner.go

```diff
@@ -31,12 +33,41 @@ type DesktopUsersProcessesRunner struct {
 	executablePath string
 }
 
+func (r *DesktopUsersProcessesRunner) addProcessForUser(uid string, osProcess *os.Process) error {
```

Comment:
```suggestion
// addProcessForUser adds process information to the internal tracking state
func (r *DesktopUsersProcessesRunner) addProcessForUser(uid string, osProcess *os.Process) error {
```

If I read this correct, it's adding the process info to the `DesktopUsersProcessesRunner` internal state? If so, a comment might help